pkg/operation/botanist/monitoring.go

```diff
@@ -541,3 +582,14 @@ func (b *Botanist) DeleteSeedMonitoring(ctx context.Context) error {
 
 	return kutil.DeleteObjects(ctx, b.K8sSeedClient.Client(), objects...)
 }
+
+// ObservedComponents is a struct from all observed pods
+type ObservedComponents struct {
+	ObservedPods []ObservedPods `yaml:"observedPods"`
+}
+
+// ObservedPods holds the Pods which are monitored through dashboards
+type ObservedPods struct {
+	PodPrefix   string `yaml:"podPrefix"`
+	UserExposed bool   `yaml:"userExposed"`
```

Comment:
The following seems better to me.

```suggestion
	IsExposedToUser bool `yaml:"isExposedToUser"`
```